src/models/constants.js

```diff
@@ -1,3 +1,3 @@
 /* eslint-disable import/prefer-default-export */
 
-export const APIURL = document.location.href.includes('search.test.datacite.org') ? 'https://api.test.datacite.org' : 'https://api.datacite.org';
+export const APIURL = document.location.href.includes('commons.test.datacite.org') ? 'https://api.test.datacite.org' : 'https://api.datacite.org';
```

Comment:
@kjgarza and @richardhallett  Feel free to check my logic here.

Not specific to this PR, but this looks this vue component would only work in production when embedded within a `search` and now `commons` page.  Otherwise it will always pull data from the test API

Reply:
Absolutely.  My logic was backwards.  Sorry about that.

I need to wait until after my first coffee before commenting on PRs.